libs/blog/videos/ui-video-dialog/src/lib/components/video-dialog/video-dialog.component.scss

```
@@ -0,0 +1,14 @@
+.dialog {
+  animation: fadeInBottom 0.4s ease-out forwards;
+}
+
+@keyframes fadeInBottom {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeInBottom" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)